pandas_datareader/yahoo/daily.py

```diff
@@ -79,8 +92,30 @@ def read(self):
             df['Ret_Index'] = _calc_return_index(df['Adj Close'])
         if self.adjust_price:
             df = _adjust_prices(df)
+        temp = pd.date_range(self.start, self.end, None, self.interval)
         return df
 
+    def _get_crumb(self, retries):
+        # Scrape a history page for a valid crumb ID:
+        tu = "https://finance.yahoo.com/quote/{}/history".format(self.symbols)
+        response = self._get_response(tu,
+                                      params=self.params, headers=self.headers)
+        out = str(self._sanitize_response(response))
+        # Matches: {"crumb":"AlphaNumeric"}
+        regex = re.search(r'{"crumb" ?: ?"([A-Za-z0-9.]{11,})"}', out)
```

Comment:
Good note, I'll push an update. All of my test cases found the crumbs nested in brackets but that's not very sustainable.